MODULE.bazel

```diff
@@ -11,18 +11,20 @@ bazel_dep(
     name = "platforms",
     version = "0.0.8",
 )
+bazel_dep(
+    name = "rules_java",
+    version = "7.3.2",
```

Comment:
I would recommend staying on a lower version as this one requires Bazel 7. 7.2.0 still works with Bazel 6.4.0.

Reply:
We already use `use_repo_rule`, so we're already expecting Bazel 7 when people use `bzlmod`